tests/utils.py

```diff
@@ -376,6 +380,9 @@ def create_tokens(manager: 'HathorManager', address_b58: Optional[str] = None, m
     script = P2PKH.create_output_script(address)
 
     deposit_amount = get_deposit_amount(mint_amount)
+    if nft_data:
+        # NFT creation needs 0.01 HTR of fee
+        deposit_amount += 1
```

Comment:
With the NFT creation we need HTR for 2 different reasons, (i) the deposit HTR for creating the new token amount and (ii) the 0.01 HTR fee to be used as the DataScript output.

This `deposit_amount += 1` is for the 0.01 HTR fee and maybe the name is misleading but I am just using this to select the utxos and calculate the change output considering the correct amount of HTR that will be "lost" in this transaction

Reply:
Oh! Let me see if I understand:

- `deposit_amount` is burnt/locked by the mint
- `0.01 HTR` is used to create an utxo (change output?) that has an unspendable script (which is why it will be "lost") that has the nft data

Does that make sense?

Reply:
Yes, exactly.